src/modules/mana/sagas.spec.ts

```diff
@@ -0,0 +1,178 @@
+import { select } from '@redux-saga/core/effects'
+import { getNetworkProvider } from 'decentraland-dapps/dist/lib/eth'
+import { getAddress } from 'decentraland-dapps/dist/modules/wallet/selectors'
+import { expectSaga } from 'redux-saga-test-plan'
+import * as matchers from 'redux-saga-test-plan/matchers'
+import {
+  importWithdrawalFailure,
+  importWithdrawalRequest,
+  importWithdrawalSuccess,
+} from './actions'
+import { handleImportWithdrawalRequest, importWithdrawalErrors } from './sagas'
+import { getMaticPOSClient } from './utils'
+import data from './sagas.spec.data.json'
+
+jest.mock('decentraland-dapps/dist/lib/eth', () => ({
+  //@ts-ignore
+  ...jest.requireActual('decentraland-dapps/dist/lib/eth'),
+  getChainIdByNetwork: jest.fn().mockReturnValue(80001),
+}))
+
+jest.mock('./utils', () => ({
+  //@ts-ignore
+  ...jest.requireActual('./utils'),
+  getMaticPOSClient: jest.fn().mockResolvedValue({
+    isERC20ExitProcessed: jest.fn().mockResolvedValue(false),
+  }),
+}))
+
+const mockGetMaticPOSClient = getMaticPOSClient as jest.Mock
+
+describe('manaSaga', () => {
+  describe('handleImportWithdrawalRequest', () => {
+    const handleImportWithdrawalRequestTest = ({
+      txHash,
+      address,
+      expectedAction,
+      sendResponse,
+    }: {
+      txHash: string
+      address: string
+      expectedAction: any
+      sendResponse?: { input: string; from: string }
+    }) =>
+      expectSaga(handleImportWithdrawalRequest, importWithdrawalRequest(txHash))
+        .provide([
+          [select(getAddress), address],
+          [
+            matchers.call.fn(getNetworkProvider),
```

Comment:
It would be interesting to see if we're calling `getNetworkProvider` with the correct parameters. If we match it independently of them, we won't be able to verify that.

Reply:
Done